lib/avro/builder/file_handler.rb

```diff
@@ -22,14 +22,14 @@ def read_file(name)
       private
 
       def find_file(name)
-        file_name = "#{name.to_s.sub(/\.rb$/, '')}.rb"
+        file_name = "/#{name.to_s.gsub('.', '/').sub(/^\//, '').sub(/\.rb$/, '')}.rb"
```

Comment:
we're doing enough subbing that a comment describing the overall transformation would be useful